geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/ListCommand.scala

```diff
@@ -0,0 +1,80 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.kafka.commands
+
+import com.beust.jcommander.{JCommander, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import kafka.utils.{ZKStringSerializer, ZkUtils}
+import org.I0Itec.zkclient.ZkClient
+import org.I0Itec.zkclient.exception.ZkNoNodeException
+import org.locationtech.geomesa.tools.common.commands.Command
+import org.locationtech.geomesa.tools.kafka.commands.ListCommand._
+import org.locationtech.geomesa.tools.kafka.{DataStoreHelper, OptionalZkPathParams}
+
+class ListCommand(parent: JCommander) extends Command(parent) with LazyLogging {
```

Comment:
zkPath is optional, in which case a KDS isn't used. If no zkPath is specified, the list command searches the entirety of zookeeper for potential geomesa schemas.